lib/methwhy.g

```diff
@@ -89,12 +89,89 @@ end);
 ##      also for each not applicable method all reasons why it is not
 ##      applicable.
 ##  </Item>
-##  <Mark>6</Mark>
-##  <Item>
-##      also the function body of the selected method(s)
-##  </Item>
 ##  </List>
 ##  <P/>
+##  Since <C>ApplicableMethod</C> returns a function, <C>Print(last);</C> 
+##  may be used to view the code. 
+##  <P/>
+##  If the first argument is a function, rather than an operation, 
+##  then the function is returned. 
+##  <P/>
+##  The first example shows that there are 50 methods for <C>Size</C> 
+##  currently available, or which 3 are applicable to <C>s4</C>. 
+##  <Example><![CDATA[
```

Comment:
I would change this Example into a Log, just to avoid having to continuously update it as code moves around (Log is like Example, except it isn't checked by make manualexamples)

Reply:
Ah - I'm not particularly aware of "make manualexamples", but wondered whether this was liable too happen.  Will change.

Reply:
Now that these are <Log> ... </Log>'s, is it better to use "GAPROOT" rather than "... some path ..."?